app/src/main/java/com/bogdan/codeforceswatcher/fragment/UsersFragment.kt

```diff
@@ -54,7 +54,7 @@ class UsersFragment : android.support.v4.app.Fragment(), SwipeRefreshLayout.OnRe
         activity?.let {
```

Comment:
You don't need an activity to use `findViewById`, you just need to move all these binding code to `onCreateView` method.

Example:
```
override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val view = inflater.inflate(R.layout.fragment_recycler, container, false)

        fab = findViewById(R.id.fab)
        
        return view
    }
```